### ISSUE TITLE
Annotation labels should not (or not only) appear in the center of the annotation

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-qt-browser#210]
**Is your feature request related to a problem? Please describe.**
I have some annotations that span several dozens of seconds. When I spot such an annotated segment in my data, there is no simple way for me to figure out the annotation description: I have to cautiously keep scrolling right until I hit the center of the annotation, because the label is centered.

**Describe the solution you'd like**
The label should be anchored to the annotation onset.

**Describe alternatives you've considered**
Another thing I'd love to see is a tooltip that displays the annotation label on hover.

**Comment by larsoner** [comment]
I'm fine with moving it to the start. 

A hover tooltip could be neat, too. Based on a very quick look it seems like it might be doable with `hoverable=True` and/or `hoverEvent`.

**Comment by cbrnr** [comment]
> I'm fine with moving it to the start.

This doesn't solve the problem though. What if you are in the middle or at the end of a long annotation? I think the label should always be visible (per page).

**Comment by hoechenberger** [comment]
> > I'm fine with moving it to the start.
> 
> This doesn't solve the problem though. What if you are in the middle or at the end of a long annotation? I think the label should always be visible (per page).

Oh that would be neat, yes

**Comment by drammock** [comment]
> This doesn't solve the problem though. What if you are in the middle or at the end of a long annotation? I think the label should always be visible (per page).

agreed, it should be centered on the *visible* portion of the annotation, not its full duration.
